### DISCUSSION TITLE
user chat history

### DISCUSSION THREAD

**Discussion by abbasudo** [tulir/whatsmeow#186] [General]
How to get chat history with someone by his jid ?

**Comment by Ased2235** [comment]
As of now, it's not possible but you can read #33

**Reply by sumnerevans** [reply]
To clarify, it's not possible because WhatsApp doesn't make it possible. It's not a bridge limitation.

**Comment by abbasudo** [comment]
@Ased2235 @sumnerevans thanks for letting me know.

**Comment by abbasudo** [comment]
is there any way to send chathistory sync requests manually?

**Reply by sumnerevans** [reply]
No.
